src/msg/async/rdma/RDMAStack.cc

```diff
@@ -453,9 +451,9 @@ int RDMAWorker::listen(entity_addr_t &sa, const SocketOptions &opt,ServerSocket
 
 int RDMAWorker::connect(const entity_addr_t &addr, const SocketOptions &opts, ConnectedSocket *socket)
 {
-  global_infiniband->init();
+  stack->get_infiniband()->init();
 
-  RDMAConnectedSocketImpl* p = new RDMAConnectedSocketImpl(cct, global_infiniband.get(), get_stack()->get_dispatcher(), this);
+  RDMAConnectedSocketImpl* p = new RDMAConnectedSocketImpl(cct, stack->get_infiniband(), get_stack()->get_dispatcher(), this);
```

Comment:
it is a coding style. Either use get_stack() or stack-> and get rid of get_stack() accessor